tools/packaging/packaging.psm1

```diff
@@ -1519,7 +1521,7 @@ function New-UnifiedNugetPackage
     {
 
         $refBinPath = New-TempFolder
-        $SnkFilePath = Join-Path $PSScriptRoot -ChildPath '../../src/signing/visualstudiopublic.snk' -Resolve
+        $SnkFilePath = (Resolve-Path "$RepoRoot/src/signing/visualstudiopublic.snk").Path
```

Comment:
I was trying to be safe just in case that it expects the path separator to be platform specific.
Will look into how this path is used and see if it really needs to be resolved.

Reply:
The path is only used in `New-ReferenceAssembly`, which only runs on windows platform. So I can change it to use `\` directory separator.